### ISSUE TITLE
tarantoolctl tar

### ISSUE THREAD

**Issue by kostja** [tarantool/tarantool#3212]
Create a tarantoolctl command which creates a Tarantool tar file with an application and all its dependencies.
It should download, compile and install all rocks/dependencies into .local directory of the project, and pack it into an installable tarball. It should download and install the specific versions of the applications written in the requirements file.

Yes, it violates the /usr/share way of shipping Lua applications (the standard way), but it should make it easy to build, ship and version applications and its dependencies.

In other words, tarantool should node.js, Java and Go path in package dependency evolution.

@todo: augment this with application evolution

See also: https://github.com/tarantool/tarantool/issues/2848

**Comment by Totktonada** [comment]
Are not this issue is duplicate of #2846?

**Comment by Totktonada** [comment]
It looks quite similar to what we have in `cartridge pack` now and what we'll have in `tt pack`. I think this issue is not more actual.
